.github/workflows/rcBuild.yml

```diff
@@ -1,57 +1,72 @@
-name: RC
+name: RC #TODO: RENAME and update failures
 on:
   release:
     types: [published]
-
-# TODO: This will need channel changes for patches/prereleases!
+    # TODO: We might need `prereleased` here
 
 jobs:
+  get-channel:
+    runs-on: ubuntu-latest
+    outputs:
+      channel: steps.release-channel.outputs.group1
+      s3-channel: steps.s3-release-channel.outputs.s3-channel
+    steps:
+      - name: Get release channel Github release
+        id: release-channel
+        uses: actions-ecosystem/action-regex-match@9e6c4fb3d5e898f505be7a1fb6e7b0a278f6665b
+        with:
+          text: ${{ github.event.release.body }} # TODO: Confirm this
```

Comment:
what would happen if someone put a number in it?

Reply:
should there be a check that a value was found in the release body?  If not, throw something?  (otherwise, everything downstream of it would need a null-check?  Or is gha going to send an empty string and everyone needs to check for empty)